config/yaml.go

```diff
@@ -41,11 +40,8 @@ type yamlConfigProvider struct {
 }
 
 var (
-	_envSeparator = []byte(":")
-	_emptyString  = []byte(`""`)
-
-	// ${VAR_NAME:default} or ${VAR_NAME}
-	_interpolationRegex = regexp.MustCompile(`\$\{\w+:?[^}]*}`)
+	_envSeparator = ":"
+	_emptyDefault = `""`
```

Comment:
Do we need to have it explicitly? We can simply use ${PATH}, instead of ${PATH:""}